utils.js

```diff
@@ -47,7 +47,7 @@ async function parseFile(file) {
 
     for (const testsuite of testsuites) {
         if(!testsuite || !testsuite.testcase) {
-            return { count, skipped, annotations };
+            continue;
```

Comment:
This causes an early return if one of the test suites doesn't contain any test cases. So now we will be able to go through all the test suites